### PR TITLE
jdk15 Implement java.lang.Access.stringConcatMix

### LINKED ISSUE #9722: jdk15 build failure java/lang/Access missing stringConcatMix()
https://ci.eclipse.org/openj9/job/Pipeline-OpenJDK-Acceptance/254/
```
[2020-05-28T02:34:59.059Z] /home/jenkins/workspace/Build_JDKnext_ppc64_aix_OpenJDK/build/aix-ppc64-server-release/support/j9jcl_sources/java.base/share/classes/.java:61: error: Access is not abstract and does not override abstract method stringConcatMix(long,String) in JavaLangAccess
[2020-05-28T02:34:59.059Z] final class Access implements JavaLangAccess {
[2020-05-28T02:34:59.059Z]       ^
```

### PR DESCRIPTION
**Pull request by pshipton**
Fixes #9722

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins test sanity aix,xlinux,xlinuxxl,xlinuxcm,plinux,zlinux,win,osx jdknext depends ibmruntimes/openj9-openjdk-jdk#openj9-staging

**Comment by pshipton** [comment]
The PR build passed, although you need to look at the following to confirm it. The openj9-staging branch (8c84c9893ba) needs to be pushed at the same time this is merged.
https://ci.eclipse.org/openj9/job/Test_openjdkNext_j9_sanity.functional_ppc64_aix_Personal_testList_0/1
https://ci.eclipse.org/openj9/job/Test_openjdkNext_j9_sanity.functional_ppc64_aix_Personal_testList_1/1

File: jcl/src/java.base/share/classes/java/lang/Access.java [hunk 1]
```diff
@@ -413,16 +413,20 @@ public ProtectionDomain protectionDomain(Class<?> clazz) {
 		return clazz.getProtectionDomain();
 	}
 
+	public Object classData(Class<?> clazz) {
+		return null;
+	}
+
 	public MethodHandle stringConcatHelper(String arg0, MethodType type) {
 		return StringConcatHelper.lookupStatic(arg0, type);
 	}
```
Comment (keithc-ca) [inline jcl/src/java.base/share/classes/java/lang/Access.java:423]: While you're tidying up, can you remove the trailing tab on this line?

Comment (pshipton) [reply jcl/src/java.base/share/classes/java/lang/Access.java]: Removed it.

**Pull request merged**